Lesson-2-Task-3/src/main/resources/archetype-resources/src/main/java/App.java

```diff
@@ -0,0 +1,13 @@
+package $org.example;
+
+/**
+ * Hello world!
+ *
+ */
+public class App 
```

Comment:
skad to sie tu wzielo? ;>
Generalnie cala zawartosc resources mozesz usunac.

Reply:
ok